about.html

```diff
@@ -0,0 +1,178 @@
+        <ul class="list">
+          <li class="mobile close">
+            <svg
+              width="40"
+              height="40"
+              viewBox="0 0 24 24"
+              fill="black"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z"
+                fill="#212121"
+              />
+            </svg>
+          </li>
+          <li><a href="#" class="link link2 about">About</a></li>
+          <li><a href="#program" class="link link2">Program</a></li>
+          <li><a href="#partner" class="link link2">Sponsor</a></li>
+          <li><a href="#" class="link link2">News</a></li>
+          <li><a href="#" class="link link2 register">Register</a></li>
+        </ul>
+      </div>
+
+      <div class="popup"></div>
+    </header>
+
+    <section class="top-section dark aboutpg">
+      <div class="hamburger mobile">
+        <div class="bar1 bar"></div>
+        <div class="bar2 bar"></div>
+        <div class="bar3 bar"></div>
+      </div>
+      <h1 class="main-heading red aboutpg">Topic:Future of blockchain <br />in African Economy <div class="small-underline aboutpg"></div></h1>
+      <div class="tech-explain aboutpg">
+        <p class="opaque">
+          Every Year the herox tech hub conducts an annual lecture on the most trending
+          technology for the past year. This annual lecture is coducted by Herox in partnership with some major companies in the  chosen technological field. This year will attracts experts from all over the world to listen and discuss with industry leads on the future of blockchain in the African economy.
+        </p>
+      </div>
+
+      <p class="contact opaque">
+        Please contact us for any further questions about Herox Tech-hub
+        blockchain Technology training 2023 <br />
+        <a href="http://" class="dark">info<span>@</span>heroxtech-hub.com</a>
+      </p>
+    </section>
+    <section class="About-logo">
+      <div>
+        <h3 class="opaque">
+          Herox Tech-hub <br />
+          Blockchain Lecture 2023
+        </h3>
+        <div class="small-underline aboutpg"></div>
+      </div>
+      <p class="logo-p mobile opaque">
+        The chosen logo for this years technological training was designed by
+        Josh Anderson. Our 2022 UI/UX trained student.
+      </p>
+      <p class="logo-p desktop opaque">
+        The chosen logo for this years technological training was designed by
+        Josh Anderson. Our 2022 UI/UX trained student. A panel was set up to
+        access logo designs from our trainees all over 50 countries. After a
+        series of screening.The panel arrived at this logo because of its
+        classic look and simplicity. After consultation with Mr.Josh we
+        optimized the price winning logo for this years blockchain training as
+        following:
+      </p>
```

Comment:
- [ ] Please `center` this `title` and `paragraph text`, not only because it is required in the project guidelines but I think it will also contribute to the beauty of the design 

<img width="996" alt="1" src="https://user-images.githubusercontent.com/77843050/210405678-965b9f44-7676-45cb-8ca2-f6b0c22564e0.png">